lib/pymedphys/_dicom/header.py

```diff
@@ -147,3 +148,96 @@ def adjust_RED_by_structure_name_cli(args):
     new_dicom_dataset = adjust_RED_by_structure_name(dicom_dataset)
 
     pydicom.write_file(args.output_file, new_dicom_dataset)
+
+
+def pretty_patient_name(
```

Comment:
May belong elsewhere? This seemed most logical of the existing files at first glance.

Reply:
Maybe a new file within:

https://github.com/pymedphys/pymedphys/tree/main/lib/pymedphys/_dicom/utilities

Something like display? I'm not sure. I'm happy for where you think best :slightly_smiling_face:.